app/Decsys/Repositories/Mongo/WebhookRepository.cs

```diff
@@ -44,7 +44,7 @@ public List<WebhookModel> List(int surveyId)
                 {
                     SurveyId = x.SurveyId,
                     CallbackUrl = x.CallbackUrl,
-                    SecretHash = x.SecretHash,
+                    Secret = x.Secret,
```

Comment:
We should never ever be revealing the secret again once set.

I'll create a separate PBI to fix that.